src/test/rpc/OwnerInfo_test.cpp

```diff
@@ -0,0 +1,216 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2017 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+#include <test/jtx.h>
+
+#include <ripple/beast/unit_test.h>
+#include <ripple/protocol/AccountID.h>
+#include <ripple/protocol/JsonFields.h>
+#include <ripple/protocol/STAmount.h>
+
+namespace ripple {
+
+class OwnerInfo_test : public beast::unit_test::suite
+{
+    void
+    testBadInput ()
+    {
+        testcase ("Bad input to owner_info");
+
+        using namespace test::jtx;
+        Env env {*this};
+
+        auto const alice = Account {"alice"};
+        env.fund (XRP(10000), alice);
+        env.close ();
+
+        { // missing account field
+            auto const result =
+                env.rpc ("json", "owner_info", "{}") [jss::result];
+            BEAST_EXPECT (result[jss::error] == "invalidParams");
+            BEAST_EXPECT (result[jss::error_message] ==
+                "Missing field 'account'.");
+        }
+
+        { // ask for empty account
+            Json::Value params;
+            params[jss::account] = "";
+            auto const result = env.rpc ("json", "owner_info",
+                boost::lexical_cast<std::string>(params)) [jss::result];
```

Comment:
So much simpler to just use `to_string(params)` here and the other places you use `lexical_cast`.